### PR TITLE
Add Malayalam (ml) translation (thank you @abrahamparayil)

### PR DESCRIPTION
**Pull request by abrahamparayil**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by marchersimon** [comment]
Thanks @abrahamparayil. We decided not to translate `example` itself, since it well be replaced by a command, so could you please "untranslate" it?

File: contributing-guides/alias-pages.md [hunk 1]
```diff
@@ -115,7 +115,15 @@ Not translated yet.
...
-Not translated yet.
+```Markdown
+# ഉദാഹരണം
+
+> ഈ കമാൻഡ് `ഉദാഹരണം` എന്നത്തിന്റെ അപരനാമമാണ്.
+
+- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:
+
+`tldr ഉദാഹരണം`
+```
```
Comment (bl-ue) [inline contributing-guides/alias-pages.md]: @marchersimon can you apply this suggestion?
```suggestion
```markdown
# example

> ഈ കമാൻഡ് `example` എന്നത്തിന്റെ അപരനാമമാണ്.

- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:

`tldr example`
```
```

Comment (marchersimon) [reply contributing-guides/alias-pages.md]: Already done (85dd9a2)

**Comment by marchersimon** [comment]
Thanks @abrahamparayil

**Pull request merged**